Neos.ContentRepositoryRegistry/Classes/ContentRepositoryRegistry.php

```diff
@@ -103,13 +80,23 @@ public function subgraphForNode(Node $node): ContentSubgraphInterface
      * @return T
      * @throws ContentRepositoryNotFoundException | InvalidConfigurationException
      * @template T of ContentRepositoryServiceInterface
+     * @deprecated in favor of {@see self::buildService()}
```

Comment:
we should add when it will be removed.

I get that we have some kind of stable api right now so @ahaeslich can rest in peace but i rather not have this actually released with 9.0

```suggestion
     * @deprecated in favor of {@see self::buildService()} will be removed with the 9.0 release
```

Reply:
and this is breaking as is, as the service is no flyweight anymore so we might as well remove the shallow method.

Reply:
I agree that we should always attach a version to all deprecation annotations, but it should be the version when it was deprecated not when it will be removed (because that has turned out not to be true more often than not)
```suggestion
     * @deprecated with version 9.0 in favor of {@see self::buildService()}
```

Reply:
Hmm 🤔 I remember @mficzel saying it’s helpful for when scanning the code before a release, but it’s not not consequently so we missed stuff

Reply:
yes, that's true in a perfect world

Reply:
But in this case as I wrote in slack I would argue to just remove the old method

https://neos-project.slack.com/archives/C3MCBK6S2/p1688734888402609